README.md

```diff
@@ -16,3 +16,39 @@ This Operator is in the early stages of implementation. For the time being, plea
    ```
 2. Make sure to uncomment the `image` in `config/manager/kustomization.yaml` and set it to the operator image you pushed
 3. Run `oc apply -k config/default`
+
+### OperatorHub install with custom index image
+
+This process refers to building the operator in a way that it can be installed locally via the OperatorHub with a custom index image.
+
+1. Build and push the bundle image to a registry:
+   ```sh
+   $ podman build -t <registry>/<username>/external-dns-operator-bundle:latest -f Dockerfile.bundle .
+   $ podman push <registry>/<username>/external-dns-operator-bundle:latest
+   ```
+
+2. Build and push the image index for operator-registry:
+   ```sh
+   # to get opm: https://github.com/operator-framework/operator-registry
```

Comment:
"to get opm" -> "download opm if needed from"

Reply:
changed